src/Orc.NuGetExplorer/Orc.NuGetExplorer.Shared/Models/PackageDetails.cs

```diff
@@ -83,6 +88,23 @@ public string Dependencies
 
         public bool? IsInstalled { get; set; }
 
+        public IList<string> AvailableVersions
+        {
+            get
+            {
+                if (_availableVersions == null)
+                {
+                    _availableVersions = _availableVersionsEnumeration.OrderByDescending(version => 
+                        version.Contains("-")
+                            ? version.Replace("-unstable", "-1").Replace("-beta", "-3").Replace("-alpha", "-2") : 
+                            version + "-4").Take(200).ToList();
```

Comment:
Please add the 200 to a static environment class or something like that. If we want to do it technically correct, it should be a separate service that gets injected. We might want to only show the most recent 10 in our apps.